notebooks/data-sources/github/thoth_PR_EDA.ipynb

```diff
@@ -0,0 +1,1892 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.                                    "Stars": repo.stargazers_count,
</pre><p>Since <code>stars</code> and <code>last_updated_at</code>  are repo-level metrics and not issue-level metrics, could we drop them from here and have them in the above df (<code>df_github.csv</code> ) instead? Otherwise there's a lot of repeating rows with the same value (for stars and last_updated_at)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/490/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>